### DISCUSSION TITLE
Multilingual support

### DISCUSSION THREAD

**Discussion by mikewink** [duncanmcclean/guest-entries#68] [Help]
Hello, 

I am using Guest Entries successfully on a one language project. Now, we are extending the project to multiple languages. 

How can I use Guest Entries in that context? How can I let it know in what language to create the new entry? Does it do this automatically by checking the current active language?

Thank you very much for any help.

**Comment by duncanmcclean** [comment]
Guest Entries should automatically guess the site based on the page you came from.

However, you can optionally tell Guest Entries which site an entry should be created in by providing a `site` input:

```html
<input type="hidden" name="site" handle="fr">
```
